initial_joint_positions:
  ros__parameters:
    panda_joint1: 0.0
    panda_joint2: -0.785
    panda_joint3: 0.0
    panda_joint4: -2.356
    panda_joint5: 0.0
    panda_joint6: 1.571
    panda_joint7: 0.785
    panda_finger_joint1: 0.04
    panda_finger_joint2: 0.04
